src/google/scraper.js

```diff
@@ -54,6 +54,15 @@ class GoogleScraper {
     await page.goto(query, {
       waitUntil: 'networkidle0',
     });
+    
```

Comment:
Checks for a accept all cookies button, if it exists press the button and wait for the page to load. Then continue the code